examples/analytics.amp.html

```diff
@@ -442,6 +442,39 @@
 </script>
 </amp-analytics>
 <!-- End SOASTA mPulse example -->
+
+<!-- Adobe Analytics example -->
+<!--
+For complete documentation and additional examples please see: https://marketing.adobe.com/resources/help/en_US/sc/implement/accelerated-mobile-pages.html
+-->
+<amp-analytics type=adobeanalytics>
+<script type="application/json">
+{
+  "requests": {
```

Comment:
Do you mean the items I have listed in the vars below? Yeah I know that extraUrlParams can be used here. In fact I'm a big fan of the extraUrlParams. Historically, variable expansion has been a little dodgy in the extraUrlParams; is that what your PR addresses?